lib/src/pages/misc/crop.dart

```diff
@@ -0,0 +1,227 @@
+import 'dart:math';
```

Comment:
At the top of the demo file, please keep author info in comments, copy format from other demos, use your info.

Reply:
I've added my author information.